static/src/js/components/AdvancedSearchModal/RegionSearchForm.js

```diff
@@ -0,0 +1,250 @@
+                      placeholder={this.getEndpointData(endpoint).placeholder}
+                      onBlur={handleBlur}
+                      onChange={handleChange}
+                      value={keyword}
+                      onKeyPress={this.handleKeypress}
+                      isInvalid={keywordErrors && keywordTouched}
+                    />
+                    {
+                      (keywordErrors && keywordTouched) && (
+                        <Form.Control.Feedback type="invalid">
+                          {keywordErrors}
+                        </Form.Control.Feedback>
+                      )
+                    }
+                  </Col>
+                </Form.Group>
+                <Row>
+                  <Col>
+                    <Row className="align-items-center">
+                      <Col>
+                        <Form.Group controlId="exact" className="mb-0">
+                          <Form.Check
+                            name="exact"
+                            type="checkbox"
+                            label="Exact match"
+                            onChange={handleChange}
+                            value={exact}
+                          />
+                        </Form.Group>
+                      </Col>
+                      <Col sm="auto">
+                        <Button
+                          label="Search"
+                          variant="full"
+                          bootstrapVariant="light"
+                          disabled={!isValid}
+                          onClick={handleSubmit}
+                          type="button"
+                        >
+                          Search
+                        </Button>
+                      </Col>
+                    </Row>
+                  </Col>
+                </Row>
+              </Col>
+              <Col>
+                {
+                  (endpoint === 'huc' || endpoint === 'region') && (
+                    <EDSCAlert
+                      variant="small"
+                      bootstrapVariant="light"
+                      icon="question-circle"
+                    >
+                      Find more information about Hydrological Units at
+                      {' '}
+                      <a
+                        className="link--external"
+                        target="_blank"
+                        rel="noreferrer noopener"
+                        href="https://water.usgs.gov/GIS/huc.html"
+                      >
+                        https://water.usgs.gov/GIS/huc.html
+                      </a>
+                    </EDSCAlert>
+                  )
+                }
+              </Col>
+            </Row>
+          )
+        }
+          {
+          !isEmpty(selectedRegion) && (
+            <p className="region-search__selected-region">
+              <span className="region-search__selected-region-id">{`${selectedRegion.type.toUpperCase()} ${selectedRegion.id}`}</span>
```

Comment:
I think since its an acronym, it should be capitalized in the markup. We aren't storing a capitalized version anywhere, so this seems to do the trick.